src/js/utils/SystemLogUtil.js

```diff
@@ -67,6 +67,46 @@ const SystemLogUtil = {
     }, []);
 
     return `${Config.logsAPIPrefix}/${nodeID}/logs/v1/${base}/${idArray.join('/')}${endpoint}?${paramsArray.join('&')}`;
+  },
+
+  accumulatedThrottle(callback, wait, useAccumulation = true) {
```

Comment:
Since it is _accumulated_ throttle I'd drop `useAccumulation` argument, we have a normal `throttle` for that or am I missing something?